docs/dictionary/keyword/then.lcdoc

```diff
@@ -26,20 +26,18 @@ If you want to execute a single statement, place the <then> <keyword>
 before the <statement> on the same line, as in these examples:
 
     if 1+1 = 2 then doSomethingTrite
-
-
+or
```

Comment:
There needs to be an additional blank line here to get the next bit to be formatted as code.